### DISCUSSION TITLE
How to get started with contribution?

### DISCUSSION THREAD

**Discussion by ShoveetSingh** [wagtail/gsoc#7] [General]
Respected sir/madam,
I am Shoveet Singh ,a computer science undergrad. I am currently enrolled in second year at Bengal Institute of Technology .I am new to open source   contributions but i am well aware of Java-script and its backend framework such as Node.js and libraries such as React.js.
I would love to contribute to your organisation but could you please tell me how to get started?

Hoping to hear from you soon.
Regards
Shoveet.

**Comment by thibaudcolas** [comment]
Hi @ShoveetSingh 👋  welcome! At this stage we expect people to go through our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md). This has the list of contribution steps we want people to take to demonstrate their understanding of what it takes to successfully participate in GSoC.
